catroid/src/main/java/org/catrobat/catroid/utils/NumberFormats.java

```diff
@@ -40,15 +40,14 @@ public static String stringWithoutTrailingZero(String value) {
 		return value;
 	}
 
-	public static String humanFriendlyFormattedShortNumber(final int number) {
-		if (number < 1_000) {
-			return Integer.toString(number);
-		} else if (number < 10_000) {
-			return Integer.toString(number / 1_000) + (number % 1_000 > 100 ? "."
-					+ Integer.toString((number % 1_000) / 100) : "") + "k";
-		} else if (number < 1_000_000) {
-			return Integer.toString(number / 1_000) + "k";
+	public static String toMetricPostfixNotation(int number) {
+		if (number >= 1_000_000) {
+			return number / 1_000_000 + "M";
+		} else if (number >= 10_000) {
```

Comment:
Unfortunately, this has not the same logic...
Prev:
1900 => 1.9k
10900 => 10k
New:
1900 => 1k
10900 => 10.9k

PS: Kotlin would be very nice in this case 😄

Reply:
Oops, I must have missed that, will fix it (and probably convert it to kotlin)

Reply:
Done